src/inabox/inabox-viewport.js

```diff
@@ -433,16 +433,30 @@ export class ViewportBindingInabox {
     }
   }
 
+  /** @override */
+  getScrollWidth() {
+    // Get actual width of document body, regardless of iframe size.
+    return this.getBodyElement()./*OK*/offsetWidth;
```

Comment:
nitpick: `this.getScrollingElement().offsetWidth`

Reply:
Done.